buildscripts/build-vs2017.cmd

```diff
@@ -45,5 +45,7 @@ GOTO test
 :test
 %UserProfile%\.nuget\packages\nunit.consolerunner\3.6.1\tools\nunit3-console.exe src/Castle.Windsor.Tests/bin/%Configuration%/net45/Castle.Windsor.Tests.dll
 
-
+c:\Windows\System32\sc.exe config NetTcpPortSharing start= demand
```

Comment:
I'd prefer this to be moved to appveyor.yml, otherwise it'll be making changes to a developer's machine and disabling something they might want enabled. I suspect it'd also cause a UAC elevation prompt.